dev-libs/rapidxml/files/rapidxml-1.13-clang-declarations.patch

```diff
@@ -0,0 +1,31 @@
+rom 2cf47bb8fb1de71bb2d2e059c15a30ffa4ca4cd6 Mon Sep 17 00:00:00 2001
```

Comment:
Include the sources of patches and any relevant info at the top where you can? (see https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions)